doc/api/fs.md

```diff
@@ -2854,7 +2854,6 @@ The following constants are meant for use with the [`fs.Stats`][] object's
 [Caveats]: #fs_caveats
 [Common System Errors]: errors.html#errors_common_system_errors
 [FS Constants]: #fs_fs_constants_1
-[MDN-Date-getTime]: https://developer.mozilla.org/en/JavaScript/Reference/Global_Objects/Date/getTime
```

Comment:
Question: This one's my miss (https://github.com/nodejs/node/pull/13173). Did you use a tool to find it?